pub-critic/src/app/components/Tabs/Tabs.module.scss

```diff
@@ -0,0 +1,49 @@
+@import "styles/variables.scss";
+@import "styles/vw-calc.scss";
+
+.tabRow {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+
+  @media screen and (max-width: 768px) {
+    justify-content: space-around;
+    align-items: center;
+  }
+}
+
+.tab {
+  width: vw-calc(150);
+  height: vw-calc(50);
+  min-height: 48px;
```

Comment:
I guess you want this to be in `vw` as well

Reply:
:white_check_mark: